terraform/accounts/main/main.tf

```diff
@@ -6,7 +6,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 3.37.0"
```

Comment:
Why do we need to bump this version?

Reply:
@bjgill I got an error about legacy providers and the upgrade not being able to proceed before I did this. I think maybe the version bump makes it fetch the new provider version which solved the issue as a side-effect.

Reply:
Sounds reasonable. Would have been nice to have this information in the commit/PR description.

Reply:
@bjgill fair point, I've updated the PR description now.